src/pages/en/tutorial/5-astro-api/2.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: ~/layouts/TutorialLayout.astro
+title: Generate tag pages automatically
```

Comment:
```suggestion
title: Generate tag pages
```
We generate them _dynamically_, not really _automatically_, but I think leaving it at "generate" portrays that